hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -1144,6 +1186,10 @@ public void commitCompaction(String compactionInstantTime, JavaRDD<WriteStatus>
 
   /**
    * Deduplicate Hoodie records, using the given deduplication funciton.
+   *
+   * @param keys RDD of HoodieKey to deduplicate
+   * @param parallelism partitions of RDD
```

Comment:
sorry, its my bad(I only coded this). this param is not required only. can you fix that.